gulp/tasks/images.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var $ = require( 'gulp-load-plugins' )();
+var config = require( '../config' ).images;
```

Comment:
Should be `var images`, see https://nodejs.org/api/child_process.html for example of naming style.